apps/src/templates/amazonFutureEngineerEligibility/amazonFutureEngineerEligibilityForm.jsx

```diff
@@ -440,76 +414,3 @@ const styles = {
     paddingLeft: 5,
   },
 };
-
-const ShippingAddressFormGroup = ({handleChange, checkValidationState}) => {
-  const renderedStateOptions = STATES.map(state => (
-    <option key={state} value={state}>
-      {state}
-    </option>
-  ));
-
-  return (
-    <div>
-      <div>
-        By checking the box above, I consent to Amazon sharing my email address
-        and school address with its third party vendor, Corporate Imaging
-        Concepts, LLC, solely in order to fulfill my request. I understand
-        Amazon's vendor will email me a promo code to allow me to select and
-        redeem my Thank You Kit items.
-      </div>
-      <FieldGroup
```

Comment:
Could be done as a follow-up but should these parameters also be removed from [the controller](https://github.com/code-dot-org/code-dot-org/blob/651b9e0431f76670108ef405b7192f8b49a7c2b8/dashboard/app/controllers/api/v1/amazon_future_engineer_controller.rb#L22)?

Reply:
Ahh yes I was grappling with that in the description lol trying to decide if it was worth the time to do with this ticket. I can just do it now😂

Reply:
oh I missed that sentence 🤦‍♀️ feel free to do it as a follow-up if you prefer!

Reply:
Made a ticket https://codedotorg.atlassian.net/browse/ACQ-1207